serializer.rst

```diff
@@ -226,6 +226,41 @@ A service leveraging `APCu`_ (and APC for PHP < 5.5) is built-in.
             ),
         ));
 
+Enabling a name converter
+-------------------------
+
+The use of a :ref:`name converter <component-serializer-converting-property-names-when-serializing-and-deserializing>` service can be defined in the configuration using the `name_converter` serializer parameter.
```

Comment:
I think should start with some explanation as to what the current behavior is. Something like:

```rst
By default, each key in the final serialized data will match the each property
name exactly (e.g. ``firstName``). But by using a :ref:`name converter <component-serializer-converting-property-names-when-serializing-and-deserializing>`,
you can transform the property names (e.g. ``firstName`` to ``first_name``).
```